apps/api/app/agents/tools/core/retrieval.py

```
@@ -9,68 +9,271 @@
+                    "score": item.get("relevance_score", 0),
+                }
+            )
 
-class RetrieveToolsResult(TypedDict):
-    """Result from retrieve_tools function."""
+    return processed
+
+
+def _process_chroma_search_result(
+    result: List[SearchItem],
+    task_idx: int,
```

Ruff (0.14.14)

Unused function argument: `task_idx`

(ARG001)